crates/typos-cli/src/bin/typos-cli/main.rs

```diff
@@ -32,6 +32,13 @@ fn run() -> proc_exit::ExitResult {
 
     init_logging(args.verbose.log_level());
 
+    // https://github.com/console-rs/dialoguer/issues/294
+    ctrlc::set_handler(move || {
+        let _ = console::Term::stdout().show_cursor();
```

Comment:
I now see the cursor but I can't see anything I type

Reply:
That's strange, for me everything behaves normal after that change.